config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -72,30 +28,102 @@ extensions:
         name = OpenShift Ansible Release from Local Source
         EOR
         sudo cp ./openshift-ansible-local-release.repo /etc/yum.repos.d
+        basename $(ls ${tito_tmp_dir}/noarch/ | grep atomic-openshift-utils ) .rpm > /data/src/github.com/openshift/aos-cd-jobs/OPENSHIFT_ANSIBLE_BUILT_VERSION
+    - type: "script"
+      title: "install ansible atomic-openshift-utils"
+      repository: "aos-cd-jobs"
+      script: |-
+        pkg_name="origin"
+        git checkout sjb
+        echo $pkg_name > PKG_NAME
+        echo "openshift-ansible openshift-ansible-callback-plugins openshift-ansible-docs openshift-ansible-filter-plugins openshift-ansible-lookup-plugins openshift-ansible-playbooks openshift-ansible-roles" > OPENSHIFT_ANSIBLE_PKGS
+        sudo python hack/determine_install_upgrade_version.py $( cat OPENSHIFT_ANSIBLE_BUILT_VERSION )
+        source ATOMIC-OPENSHIFT-UTILS_INSTALL_UPGRADE_VERSION
```

Comment:
So you suggest just to print out the versions from the script ?
Well Im not the biggest fan of both versions but I sticked to the idea to do less magic with bash and more with python scripts, thats why I'm writing it directly to file, so in other steps we just source the file and we are good.
But I can go with:
```
sudo hack/determine_install_upgrade_version.py "$( cat OPENSHIFT_ANSIBLE_BUILT_VERSION )" > OPENSHIFT_ANSIBLE_VARS
source OPENSHIFT_ANSIBLE_VARS
```